### PR TITLE
Fix URL typo and whitespace.

### PR DESCRIPTION
**Pull request by madhusudancs**
Fixes for issue #1948.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1950)

### CODE REVIEW

**Comment by nikhiljindal** [comment]
lgtm

File: docs/admin/federation/kubefed.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ assignees:
 - madhusudancs
 
 ---
+
```
Comment (janetkuo) [inline docs/admin/federation/kubefed.md]: We don't need this. Ref http://kubernetes.io/docs/contribute/page-templates/

File: docs/admin/federation/kubefed.md [hunk 2]
```diff
@@ -61,8 +60,8 @@ The output should contain an entry corresponding to your host cluster,
 similar to the following:
 
 ```
-CURRENT   NAME                                                               CLUSTER                                                            AUTHINFO                                                           NAMESPACE
-          gke_myproject_asia-east1-b_gce-asia-east1              gke_myproject_asia-east1-b_gce-asia-east1              gke_myproject_asia-east1-b_gce-asia-east1
+CURRENT   NAME                                          CLUSTER                                       AUTHINFO                                      NAMESPACE
+          gke_myproject_asia-east1-b_gce-asia-east1     gke_myproject_asia-east1-b_gce-asia-east1     gke_myproject_asia-east1-b_gce-asia-east1
```
Comment (janetkuo) [inline docs/admin/federation/kubefed.md]: I still can't scroll this part. Maybe it's a website issue that needs to be fixed by @kubernetes/sig-docs?

File: _data/guides.yml [hunk 3]
```diff
@@ -306,6 +306,6 @@ toc:
 - title: Administering Federation
   section:
   - title: Using `kubefed`
```
Comment (janetkuo) [inline _data/guides.yml]: Question for @kubernetes/sig-docs: ` ` doesn't render as code block in titles, should we keep this?

File: docs/admin/federation/kubefed.md (hunk 1 above)
Comment (madhusudancs) [inline docs/admin/federation/kubefed.md]: This caused the left nav to come back, surprisingly.

File: _data/guides.yml (hunk 3 above)
Comment (jaredbhatti) [inline _data/guides.yml]: I would remove this.

Comment (madhusudancs) [inline _data/guides.yml]: @jaredbhatti @janetkuo done.

**Pull request merged**